package proto_ff;

import "yd_fieldoptions.proto";


message E_XiuzhenroadTask
{
	optional int64 m_taskId = 1[(yd_fieldoptions.field_cname) = "任务ID"];
	optional int32 m_taskType = 2[(yd_fieldoptions.field_cname) = "任务类型"];
	optional int32 m_eventType = 3[(yd_fieldoptions.field_cname) = "事件类型"];
	optional int32 m_reachPrice = 4[(yd_fieldoptions.field_cname) = "达成值"];
	optional int32 m_reward = 5[(yd_fieldoptions.field_cname) = "奖励物品"];
	optional int32 m_reward_Value = 6[(yd_fieldoptions.field_cname) = "奖励物品数量"];
	optional int32 m_boxID = 7[(yd_fieldoptions.field_cname) = "box礼包ID"];
	optional int32 m_integral = 8[(yd_fieldoptions.field_cname) = "奖励的积分"];
	optional int32 m_linkid = 9[(yd_fieldoptions.field_cname) = "跳转ID"];
	repeated int32 m_param = 10[(yd_fieldoptions.field_cname) = "事件完成参数", (yd_fieldoptions.field_arysize) = 3];
}


message Sheet_XiuzhenroadTask
{
	repeated E_XiuzhenroadTask E_XiuzhenroadTask_List = 1[(yd_fieldoptions.field_arysize)=128];
}

message E_XiuzhenroadBonuspointsTotalitemsDesc
{
	optional int32 m_Value = 1[(yd_fieldoptions.field_cname) = "数量"];
	optional int32 m_ID = 2[(yd_fieldoptions.field_cname) = "ID"];
}

message E_XiuzhenroadBonuspoints
{
	optional int32 m_totalRewardsID = 1[(yd_fieldoptions.field_cname) = "总积分奖励"];
	optional int32 m_achievedPoints = 2[(yd_fieldoptions.field_cname) = "达成点数"];
	repeated E_XiuzhenroadBonuspointsTotalitemsDesc m_totalItems = 3[(yd_fieldoptions.field_cname) = "奖励物品", (yd_fieldoptions.field_arysize) = 2];
}


message Sheet_XiuzhenroadBonuspoints
{
	repeated E_XiuzhenroadBonuspoints E_XiuzhenroadBonuspoints_List = 1[(yd_fieldoptions.field_arysize)=32];
}

message E_XiuzhenroadSkill
{
	optional int32 m_activitySkills = 1[(yd_fieldoptions.field_cname) = "活动分类技能"];
	optional int32 m_skillID = 2[(yd_fieldoptions.field_cname) = "技能ID"];
	optional int32 m_itenmID = 3[(yd_fieldoptions.field_cname) = "属性道具ID"];
	optional int32 m_attribute = 4[(yd_fieldoptions.field_cname) = "道具增加属性"];
}


message Sheet_XiuzhenroadSkill
{
	repeated E_XiuzhenroadSkill E_XiuzhenroadSkill_List = 1[(yd_fieldoptions.field_arysize)=32];
}

message E_XiuzhenroadClassify
{
	optional int32 m_typeID = 1[(yd_fieldoptions.field_cname) = "任务类型ID"];
	optional string m_gift = 2[(yd_fieldoptions.field_cname) = "礼包名称", (yd_fieldoptions.field_bufsize) = 64];
	optional int32 m_giftID = 3[(yd_fieldoptions.field_cname) = "礼包ID"];
	optional int32 m_rechargeID = 4[(yd_fieldoptions.field_cname) = "礼包计费点ID"];
}


message Sheet_XiuzhenroadClassify
{
	repeated E_XiuzhenroadClassify E_XiuzhenroadClassify_List = 1[(yd_fieldoptions.field_arysize)=16];
}
